hcm:

  start_time: 0 # time we wait for getting motor updates before leaving startup state
  soft_off_time: 30 # time of no use or updates when the hcm goes to soft off
  die_time: 60 # time of no use or updates when the hcm shuts down
  anim_server_wait_time: 10 # time the hcm waits for animation server

  animations:
    walkready: "walkready" # Gibt eine Dynamische Walkready #todo change to dynamic pose from walking, like it was before
    goalie-walkready: "walkready" # Gibt eine Dynamische Walkready #todo change to dynamic pose from walking, like it was before
    front-up: "standup-minibot-new-front-up"
    bottom-up: "standup-minibot-new-back-up"
    toSquat: "sit-down"
    stand-up: "stand-up"
    penalized: "sit-down"
    penalized_end: "stand-up"
    sit-down: "sit-down"
    shut-down: "sit-down"
    init: "init"

  falling:

    #falling animations
    "falling_front": "falling_front"
    "falling_back": "falling_back"
    "falling_left": "falling_left"
    "falling_right": "falling_right"

    "Minibot":

      #Grenzwerte m체ssen auf h채rterem boden auch h채rter eingestellt werden,
      #dies erfolgt mit dem boden_koeffizient
      "ground_coefficient": 1.0  #WEICH(kunstrasen Spielfeld)= 1.0 , HART(teppich Spielfeld)= 1.25

      #Grenzwerte sch채rfer --> fruehere Reaktion aber haeufigere Fehlinterpretation, zb. beim Laufen
      #Grenzwerte sanfter --> zu spaete Raktion
      "threshold_gyro_y_front":  -2.272 # > gyroY
      "threshold_gyro_y_back":  2.272 # < gyroY
      "threshold_gyro_x_right": -1.63 # > gyroX
      "threshold_gyro_x_left":   1.63 # < gyroX

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": true
